app/scripts/modules/netflix/ci/services/build.read.service.ts

```diff
@@ -0,0 +1,119 @@
+import {IHttpPromiseCallbackArg, IPromise, module} from 'angular';
+
+import {Api, API_SERVICE} from 'core/api/api.service';
+import {CI_FILTER_MODEL, CiFilterModel} from '../ci.filter.model';
+import {
+  ORCHESTRATED_ITEM_TRANSFORMER,
+  OrchestratedItemTransformer
+} from 'core/orchestratedItem/orchestratedItem.transformer';
+import {SETTINGS} from 'core/config/settings';
+
+interface ICiBuildChangeRef {
+  id: string;
+  displayId: string;
+}
+
+interface ICiBuildChangeTargetAuthor {
+  email: string;
+  name: string;
+  username: string;
+}
+
+interface ICiBuildChangeTarget extends ICiBuildChangeRef {
+  author: ICiBuildChangeTargetAuthor;
+  message: string;
+}
+
+interface ICiBuildChange {
+  from: ICiBuildChangeTarget;
+  ref: ICiBuildChangeRef;
+  to: ICiBuildChangeTarget;
+}
+
+export interface ICiBuild {
+  buildNumber: number;
+  change: ICiBuildChange;
+  completedAt: number;
+  completionStatus: string;
+  endTime?: number;
+  id: string;
+  isRunning?: boolean;
+  repositoryId: string;
+  startedAt: number;
+  startTime?: number;
+}
+
+export interface ICiBuildOutputConfig {
+  data: string[];
+  last: boolean;
+  nextStart: string;
+}
+
+export class CiBuildReader {
+
+  public static get MAX_LINES(): number {
+    return 4095;
+  };
+
+  static get $inject(): string[] {
+    return ['API', 'orchestratedItemTransformer', 'CiFilterModel'];
+  }
+
+  constructor(private API: Api,
+              private orchestratedItemTransformer: OrchestratedItemTransformer,
+              private ciFilterModel: CiFilterModel) {
```

Comment:
I really don't like that the injected value is using a different name than the the exported object.

Since the `CiFilterModel` class is so simple (it's just one field, no methods, nothing injected), we might as well remove it from Angular altogether so we don't need to rely on changing the case of the injected field.